lib/vector-core/core-common/src/internal_event/bytes_sent.rs

```diff
@@ -0,0 +1,23 @@
+use crate::internal_event::InternalEvent;
```

Comment:
I just realized that the `check-events` scanner won't be looking at these. Do you want to add this to the script, or leave that to me in a separate issue?

Reply:
I added it.